### ISSUE TITLE
gnt-cluster modify --no-lvm-storage broken

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#341]
Originally reported of Google Code with ID 285.

```
[cluster] root@node4:~# gnt-cluster modify --no-lvm-storage
Failure: prerequisites not met for this operation:
error type: wrong_input, error details:
Parameter 'OP_CLUSTER_SET_PARAMS.vg_name' fails validation

I have lvm-based instances, so this should fail, but not like this.
```
Originally added on 2012-10-30 11:21:33 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2012-10-30 12:18:55 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking as released the bugs fixed in 2.6.2.
```
Originally added on 2013-01-11 12:13:05 +0000 UTC.

Changed State: Released
